packages/antd/src/components/crud/edit/index.spec.tsx

```diff
@@ -1,14 +1,16 @@
-import React, { ReactNode } from "react";
+import React, { ComponentProps, ReactNode } from "react";
 import { Route, Routes } from "react-router-dom";
-import { AccessControlProvider } from "@pankod/refine-core";
+import { Refine } from "@pankod/refine-core";
 import { Button } from "antd";
 
 import { render, TestWrapper, waitFor } from "@test";
 import { Edit } from "./";
 
 const renderEdit = (
     edit: ReactNode,
-    accessControlProvider?: AccessControlProvider,
+    accessControlProvider?: ComponentProps<
+        typeof Refine
+    >["accessControlProvider"],
```

Comment:
```suggestion
    accessControlProvider?: AccessControlProvider
```

I think it would be better if you wrote it like this.

Reply:
updated the usage as suggested